stable/firehose/templates/deployment.yaml

```diff
@@ -85,3 +91,51 @@ spec:
       - emptyDir: {}
         name: workdir
       {{- end }}
+      {{- if or (gt (len .Values.volumes) 0) }}
```

Comment:
do we need this check? empty lists would automatically give the same effect right? also, what is the `or` doing